### PR TITLE
macOS fix some layout issues in torrent inspector window

### PR DESCRIPTION
**Pull request by sweetppro**
Notes: Fixed layout issues in torrent inspector window.

The initial PR #3409 had some UI issues which I missed, this PR fixes those.
### Before:

https://user-images.githubusercontent.com/7323792/207581301-1a863c2a-9f8c-456e-af3b-0fde809f69b8.mov

### This PR:

https://user-images.githubusercontent.com/7323792/207581373-fd4ab603-f152-49fd-a396-8d7aee95cc9f.mov

### CODE REVIEW

**Comment by sweetppro** [comment]
@Coeur or @nevack can you take a look at this

File: macosx/InfoActivityViewController.mm [hunk 1]
```diff
@@ -180,50 +179,68 @@ - (CGFloat)vertLayoutHeight
     return NSHeight(self.fTransferView.frame) + NSHeight(self.fDatesView.frame) + (2 * kStackViewInset) + kStackViewVerticalSpacing;
 }
 
-- (CGFloat)changeInWindowHeight
+- (void)updateOldHeight:(CGFloat)height
+{
+    self.oldHeight = height;
+}
```
Comment (Coeur) [inline macosx/InfoActivityViewController.mm:185]: Wouldn't it be `setOldHeight:` and `_oldHeight = height`?

File: macosx/InfoActivityViewController.mm [hunk 2]
```diff
@@ -180,50 +179,68 @@ - (CGFloat)vertLayoutHeight
...
 {
-    CGFloat difference = 0;
+    return self.oldHeight - self.currentHeight;
+}
 
+- (NSRect)viewRect
+{
+    CGFloat difference = self.heightChange;
+
+    NSRect windowRect = self.view.window.frame, viewRect = self.view.frame;
```
Comment (Coeur) [inline macosx/InfoActivityViewController.mm:196]: We generally prefer one assignment per line, please.

File: macosx/InfoActivityViewController.mm (hunk 1 above)
Comment (Coeur) [reply macosx/InfoActivityViewController.mm:185]: Actually, removing this method entirely and just moving `@property(nonatomic) CGFloat oldHeight;` to the header should be good.

File: macosx/InfoActivityViewController.mm (hunk 2 above)
Comment (sweetppro) [reply macosx/InfoActivityViewController.mm]: sure thing, I cleaned up some other declarations also

File: macosx/InfoActivityViewController.mm (hunk 1 above)
Comment (sweetppro) [reply macosx/InfoActivityViewController.mm:184]: no problem, Ive also updated all the other properties to fit better with the project conventions

**Comment by Coeur** [comment]
I think that you forgot to remove the two methods named `updateOldHeight:`.

**Comment by sweetppro** [comment]
> I think that you forgot to remove the two methods named `updateOldHeight:`.

oops,
yes I did.

I took the opportunity to also remove a bunch of redundant manual layout code

**Comment by sweetppro** [comment]
oops, closed PR accidentally... I thought I was closing a comment
🤦‍♂️

**Review by Coeur — APPROVED** [review]
Reviewed and tested.
Thanks.

**Comment by sweetppro** [comment]
@Coeur I just moved some of the code to #4374 as I think it better suited being in its own PR

**Review by nevack — APPROVED** [review]
Well done, @sweetppro !
LGTM

**Comment by ckerr** [comment]
Notes: macOS layout improvements.

**Pull request merged**